src/guiloop/x11.rs

```diff
@@ -440,6 +440,11 @@ impl GuiEventLoop {
     fn process_xcb_event(&self, event: &xcb::GenericEvent) -> Result<(), Error> {
         if let Some(window_id) = Self::window_id_from_event(event) {
             self.process_window_event(window_id, event)?;
+        } else { // [FIXME] can't identify window from xkb keyboard event
+            let mut windows = self.windows.borrow_mut();
+            for w in windows.by_id.values_mut() {
+                w.dispatch_event(event)?;
```

Comment:
I think this is perhaps the only potential technical issue in this PR.
Is there a possibility that we'd dispatch the same events to multiple windows?
In a quick test with opening two windows at once, my normal non-IME text seems to do the right thing, so maybe it's ok and it just feels slightly bad because it is `O(number-of-windows)`?